provider/testdata/src/bin/datagen.rs

```diff
@@ -80,4 +81,35 @@ fn main() {
         vec![json_out, blob_out, mod_out, postcard_out],
     )
     .unwrap();
+
+    icu_datagen::datagen(
+        None,
+        &[icu_provider::hello_world::HelloWorldV1Marker::KEY],
+        &source_data,
+        vec![
+            Out::Fs {
+                output_path: paths::data_root().join("../../fs/tests/data/json"),
```

Comment:
I don't see this as a binary that should be used by a client. I've actually wanted to move this out of the crate into our script directory.

Reply:
I think it used to be a cargo-make task, but then I think you moved it in here so that you could use the datagen API instead of the CLI?

Reply:
Yes because it supports outputting all three formats at the same time, before we were running multiple invocations.

I didn't at the time realise this would be public.